src/gameLoader/gameLoader.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import {Board} from "../board/board"
+import chalk from "chalk";
+import {getFilePath} from "../cli/cli";
+
+const CANNOT_READ_FILE_ERROR = "Can't read file; please check your -f argument point to a valid file.";
+
+export const loadGameConfigFromFile = ():Board => {
+    const filePath = getFilePath();
+    return parseBoardFromPath(filePath);
+}
+
+const parseBoardFromPath = (filePath:string):Board => {
+    let fileContent;
+    let board;
+
+    try{
+        fileContent = fs.readFileSync(filePath, 'utf8')
+    }
+    catch(error){
+        console.error(chalk.red(CANNOT_READ_FILE_ERROR));
+        return null;
+    }
```

Comment:
```suggestion
    try {
        fileContent = fs.readFileSync(filePath, 'utf8')
    } catch (error) {
        console.error(chalk.red(CANNOT_READ_FILE_ERROR));
        return null;
    }
```

You should configure prettier to do that for you.

Reply:
Done, also configured eslint.